src/components/Back.svelte

```diff
@@ -1,7 +1,7 @@
 <script>
   import {push, link} from 'svelte-spa-router'
   export let href;
-  export let nomargin;
+  export let nomargin = undefined;
```

Comment:
I'm not very familiar with Svelte, is `nomargin` needed anymore?
I've suppressed the following warning by "setting" `nomargin` here: `<Back> created without expected prop 'nomargin'`